### PR TITLE
changed the target sdk from 30 to 31

### PR DESCRIPTION
**Pull request by sanchi0204**
### Description

Thanks for contributing this Pull Request. Make sure that you send in this Pull Request to the `develop` branch of this repository, add a brief description, and tag the relevant issue(s) and PR(s) below.

- Relevant Issues :  #35 

- What Changed and Why? :

Changed the target sdk from 30 to 31

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (katielevy1) [inline .idea/compiler.xml:1]: Can you remove this file from the PR? Can you also add the .idea/compiler.xml to the .gitignore file?

File: truffleshuffle/build.gradle [hunk 2]
```diff
@@ -9,7 +9,7 @@ android {
```
Comment (katielevy1) [inline truffleshuffle/build.gradle:9]: can you also change the compileSDK version in this file, as well as in the app/build.gradle file

File: .idea/compiler.xml (hunk 1 above)
Comment (sanchi0204) [reply .idea/compiler.xml]: I have removed the file from PR and have also added to .gitignore file. Pls review the PR @katielevy1

File: truffleshuffle/build.gradle (hunk 2 above)
Comment (sanchi0204) [reply truffleshuffle/build.gradle:9]: I have made the changes to app/build.gradle file and have also updated the compileSDK

**Review by katielevy1 — COMMENTED** [review]
Awesome, looks good!

**Comment by katielevy1** [comment]
@all-contributors add @sanchi0204 for code

**Pull request merged**